SKILL.tsv

```diff
@@ -5234,7 +5234,7 @@ SKILL_20150918_005233	$Cloaking: Movement Speed Increase
 SKILL_20150918_005234	$Increases the movement speed of [Cloaking] by 1 per attribute level.
 SKILL_20150918_005235	
 SKILL_20150918_005236	
-SKILL_20150918_005237	$Deals 10% additional damage to Plant-type monsters when equipped with a [Blunt].
+SKILL_20150918_005237	$Deals 10% additional damage to Plant-type monsters when equipped with a [Bludgeon] type weapon.
```

Comment:
This is talking about the weapon type, I don't know if its called [Blunt] or [Blunt Instrument].